src/content/fr/2019/cms.md

```diff
@@ -0,0 +1,662 @@
+---
+part_number: III
+chapter_number: 14
+title: CMS
+description: Chapitre CMS de l'Almanach Web 2019 couvrant l'adoption des CMS, la façon dont les solutions CMS sont construites, l'expérience utilisateur des sites web propulsés par les CMS et l'innovation des CMS.
+authors: [ernee, amedina]
+reviewers: []
+translators: [JustinyAhin]
+discuss: 
```

Comment:
Don't think we plan on having language specific discussion forums at this time, so again just leave as English one:

```suggestion
discuss: 1769
```